.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+Soundtrack of this PR: [link to song that really fits the mood of this PR]()
+
+### In this PR
+* < Additions, removals, fixes, features & functionality and the motivation for these changes, e.g. "This is needed because..." >
+
+< Ticket status, e.g. "fixes #issue number" > 
+
+### Remaining TODO
+- [ ] < Issues remaining before this PR can be merged >
```

Comment:
I'd say kill this section altogether, unfinished PRs should be drafts.

Reply:
I frequently use this format to track if I am waiting on a deploy, or waiting on another PR to get merged, even if this PR is ready for review. This template will also be automatically deployed for draft PRs.